test/upgrade/skip_at_verification_test.go

```diff
@@ -23,31 +23,31 @@ import (
 )
 
 func TestSkipAtBackgroundVerification(t *testing.T) {
+	log, buf := newExampleZap()
```

Comment:
Why all those changes to this test? It seams not required to me at all.

Reply:
The background operations were using a logger that doesn't have the right logs anymore. The buffer now has them. So I'm passing the buffer/logger directly. The goal of this test is to check that the tests are/aren't skipped.